draft-dekater-scion-dataplane.md

```diff
@@ -1089,6 +1102,8 @@ For high-speed packet processing in the **data plane**, computing even cheap ope
 
 When combining path segments to create a path to the destination endpoint, the source endpoint MUST also initialize the value of accumulator field `Acc` for each path segment. The `Acc` field MUST contain the correct XOR-sum of the path segment identifier and preceding hop field MACs expected by the first router that is traversed.
 
+The aggregated 16-bit path segment identifier and preceding MACs prevents the splicing parts of different path segments, unless there is a per-chance collision among compatible path segments in one AS. Path segments are "compatible" for splicing only if they enter the AS from the same ingress interface and have the same origination timestamp. With typical numbers of paths in
```

Comment:
>  With typical numbers of paths in 

Some part of the sentence is missing?

Reply:
Ah good catch. I had started to write the analysis on path splicing here. Shortened, and replaced it with a reference to the security analysis.